### PR TITLE
Add Delete Blob Directory Support to Activity Log

### LINKED ISSUE #1112: No logs in ACTIVITYLOG window when deleting folder in blob container.
**Repro Steps**
1. Expand a blob container 
2. Delete folder within blob container 
3. Check log message in ACITIVITYLOG window

**Expect**
"Delete 'xxx' folder Succeeded" log in the ACTIVITYLOG window

**Actual**
No message

### PR DESCRIPTION
**Pull request by motm32**
Fixes #1112

### CODE REVIEW

File: src/commands/deleteBlobDirectory/DeleteBlobDirectoryConfirmationStep.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { AzureWizardPromptStep, DialogResponses, nonNullProp } from "@microsoft/vscode-azext-utils";
+import { localize } from "../../utils/localize";
+import { IDeleteBlobDirectoryWizardContext } from "./IDeleteBlobDirectoryWizardContext";
+
+export class DeleteBlobDirectoryConfirmationStep extends AzureWizardPromptStep<IDeleteBlobDirectoryWizardContext> {
```
Comment (alexweininger) [inline src/commands/deleteBlobDirectory/DeleteBlobDirectoryConfirmationStep.ts:5]: We have a pre-made `DeleteConfirmationStep` in `utils` which should be able to save us from writing having to add this class.

https://github.com/microsoft/vscode-azuretools/blob/0b22f9315f3fd86cf3614d5100df005689ca76b6/utils/src/wizard/DeleteConfirmationStep.ts

**Review by alexweininger — COMMENTED** [review]
Small thing: the files you added are missing the required header that we put at the top of each file. 

I added the header as a VS Code snippet so I can type "header" and it will paste the whole thing in there for me. I can share my snippets file if you want.

File: src/commands/deleteBlobDirectory/DeleteBlobDirectoryConfirmationStep.ts (hunk 1 above)
Comment (motm32) [reply src/commands/deleteBlobDirectory/DeleteBlobDirectoryConfirmationStep.ts:5]: Sounds good I will add those headers and the snippets file would be super helpful. I also did see the `DeleteConfirmationStep` but was not sure how to use it along with the `suppressMessage`  if statement. But I'll try to re work it using the pre made `DeleteConfirmationStep`.

Comment (alexweininger) [reply src/commands/deleteBlobDirectory/DeleteBlobDirectoryConfirmationStep.ts:5]: Could you conditionally include the confirmation step in the prompt steps based on the value of `suppressMessage`?

Comment (motm32) [reply src/commands/deleteBlobDirectory/DeleteBlobDirectoryConfirmationStep.ts:5]: Yeah that could potentially work I will try it out.

Comment (alexweininger) [reply src/commands/deleteBlobDirectory/DeleteBlobDirectoryConfirmationStep.ts:5]: For the snippets, use the "Configure User Snippets" command and then select typescript and add this:

```
"License": {
        "prefix": [
            "license",
            "header",
            "copyright"
        ],
        "body": "/*---------------------------------------------------------------------------------------------\n*  Copyright (c) Microsoft Corporation. All rights reserved.\n*  Licensed under the MIT License. See License.txt in the project root for license information.\n*--------------------------------------------------------------------------------------------*/\n",
        "description": "Microsoft copyright license file header"
    },
```

We could actually add these to each repo so everyone has them by default. I'll look into that.

File: src/tree/blob/BlobDirectoryTreeItem.ts [hunk 2]
```diff
@@ -18,6 +20,7 @@ import { IStorageRoot } from "../IStorageRoot";
 import { BlobContainerTreeItem } from "./BlobContainerTreeItem";
 import { BlobTreeItem, ISuppressMessageContext } from "./BlobTreeItem";
 
+
```
Comment (alexweininger) [inline src/tree/blob/BlobDirectoryTreeItem.ts:23]: nit: extra newline

File: src/tree/blob/BlobDirectoryTreeItem.ts [hunk 3]
```diff
@@ -1,14 +1,16 @@
 /*---------------------------------------------------------------------------------------------
- *  Copyright (c) Microsoft Corporation. All rights reserved.
- *  Licensed under the MIT License. See License.md in the project root for license information.
- *--------------------------------------------------------------------------------------------*/
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
```
Comment (alexweininger) [inline src/tree/blob/BlobDirectoryTreeItem.ts:4]: nit: revert these changes

**Review by alexweininger — APPROVED** [review]
